### ISSUE TITLE
Show duplicated tip labels

### ISSUE THREAD

**Issue by josephwb** [OpenTreeOfLife/opentree#1117]
Recently, the "show duplicated tips" only appear if a tree is Included in synthesis. This is a bit of a bummer, since this is a good way to to doublecheck mapping: if there are duplicate mapped tips when they are not expected, then probably mapped a bad synonym.

This seems important, as a tree may be mapped in the curator, but added to synthesis by another route, such that the duplicate tips is not detected (and which of the duplicates survives to synthesis may adversely impact downstream synthesis).

**Comment by josephwb** [comment]
For my purposes, I want to use trees for something other than synthesis.

**Comment by jimallman** [comment]
> Recently, the "show duplicated tips" only appear if a tree is Included in synthesis. This is a bit of a bummer, since this is a good way to to doublecheck mapping: if there are duplicate mapped tips when they are not expected, then probably mapped a bad synonym.

Yes, things are different now. The old "preferred" flag represented **intent** to use a tree in synthesis, so we have time to clean up in response to stricter quality checks. The new include-in-synth-collections feature takes immediate action, so I think it makes sense to impose stricter quality checks up-front, for all trees.

A side effect of this is that it might complain about "unimportant" trees. Not sure if that's a practical problem or just an annoyance.

**Comment by josephwb** [comment]
Could there possibly be a dry-run "Include"? That is, run the check _before_ it is actually added to the synth collection. If it passes, can click "Confirm" to, er, confirm.

**Comment by jimallman** [comment]
We already have a minimal validator that runs (quickly) on all trees before they can be added to synthesis. It would be cleaner to add tests there.
